### PR TITLE
Add app name to kubectl gs template app

### PR DESCRIPTION
**Pull request by rossf7**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/app-platform/getting-started/index.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ CRs.
 ```nohighlight
 kubectl gs get appcatalogs
 
-NAME                    CATALOG URL                                                   CREATED
-giantswarm              https://giantswarm.github.io/giantswarm-catalog/              17 Jul 20 17:28 CEST
-giantswarm-playground   https://giantswarm.github.io/giantswarm-playground-catalog/   17 Jul 20 17:28 CEST
+NAME                    CATALOG URL
+giantswarm              https://giantswarm.github.io/giantswarm-catalog/
+giantswarm-playground   https://giantswarm.github.io/giantswarm-playground-catalog/
```
Comment (rossf7) [inline src/content/app-platform/getting-started/index.md:93]: These dates are causing wrapping.I don't think they are needed to understand what is going on. So I've just removed them.

File: src/content/app-platform/getting-started/index.md [hunk 2]
```diff
@@ -139,6 +139,10 @@ spec:
   version: 1.17.0
 ```
 
+The `--name` parameter is the name of the app in the catalog and the name of
+the App CR. The App CR name can be changed via the `--app-name` parameter which
+allows installing multiple instances of an app.
```
Comment (rossf7) [inline src/content/app-platform/getting-started/index.md:144]: Adds a note to the getting started guide.

File: src/content/ui-api/kubectl-gs/template-app.md [hunk 3]
```diff
@@ -63,6 +63,7 @@ spec:
 
 It also supports the following optional flags:
 
+- `--app-name`: Name of the App CR. Can be set when you want to change the name or install multiple instances of an app.
```
Comment (axbarsan) [inline src/content/ui-api/kubectl-gs/template-app.md:66]: Can we also mention here where the default value is coming from, please?

Comment (rossf7) [reply src/content/ui-api/kubectl-gs/template-app.md]: Yes thanks I've added.

**Review by axbarsan — APPROVED** [review]

**Review by marians — APPROVED** [review]

**Pull request merged**